### DISCUSSION TITLE
Bug in "validation" CLI 0.3.0 beta3?

### DISCUSSION THREAD

**Discussion by elementalTIMING** [elgatosf/elgato#109] [Help]
Hi Elgato Team,

the validation fails with the new CLI tool v0.3.0.

	3:13  error    Version must be semantic version with a build number (pre-release identifiers are not permitted)

**manifest.jspn**

	{
		"Version": "1.0.0",
		...

Well, I guess it is in semantic format...

Thx for fixing!

**Comment by ChekTek** [comment]
The validation is looking for a build number here. The error from validation has been improved. ✅

```
Version must use format {major}.{minor}.{patch}.{build}
```
